src/frontend/komponenter/Oppgavebenk/OppgaveRad.tsx

```diff
@@ -1,28 +1,53 @@
 import React from 'react';
 import { IOppgave } from './oppgave';
-import { oppgaveTypeTilTekst, prioritetTilTekst } from './oppgavetema';
+import { Oppgavetype, oppgaveTypeTilTekst, prioritetTilTekst } from './oppgavetema';
 import { enhetsmappeTilTekst } from './enhetsmappe';
 import { Behandlingstema, behandlingstemaTilTekst } from '../../typer/behandlingstema';
-import { useHistory } from 'react-router-dom';
 import { formaterIsoDato } from '../../utils/formatter';
-import { Flatknapp } from 'nav-frontend-knapper';
-import { useApp } from '../../context/AppContext';
-import { Ressurs, RessursStatus } from '../../typer/ressurs';
+import { Flatknapp as Knapp } from 'nav-frontend-knapper';
+import UIModalWrapper from '../Felleskomponenter/Modal/UIModalWrapper';
+import { Normaltekst } from 'nav-frontend-typografi';
+import hiddenIf from '../Felleskomponenter/HiddenIf/hiddenIf';
+import { useOppgave } from '../../hooks/useOppgave';
 
 interface Props {
     oppgave: IOppgave;
-    settFeilmelding: (melding: string) => void;
 }
 
-interface OppgaveDto {
-    behandlingId: string;
-    gsakId: string;
-}
+const Flatknapp = hiddenIf(Knapp);
+
+const kanJournalføres = (behandlingstema?: Behandlingstema, oppgavetype?: Oppgavetype) => {
+    return (
+        oppgavetype === 'JFR' &&
+        behandlingstema &&
+        ['ab0071', 'ab0177', 'ab0028'].includes(behandlingstema)
+    );
+};
+
+const kanBehandles = (behandlingstema?: Behandlingstema, oppgavetype?: Oppgavetype) => {
+    return (
+        oppgavetype === 'BEH_SAK' &&
+        behandlingstema &&
+        ['ab0071', 'ab0177', 'ab0028'].includes(behandlingstema)
+    );
+};
 
-const OppgaveRad: React.FC<Props> = ({ oppgave, settFeilmelding }) => {
+const kanTotrinnskontrolleres = (behandlingstema?: Behandlingstema, oppgavetype?: Oppgavetype) => {
+    return (
+        (oppgavetype === 'GOD_VED' || oppgavetype === 'BEH_UND_VED') &&
+        behandlingstema &&
+        ['ab0071', 'ab0177', 'ab0028'].includes(behandlingstema)
+    );
+};
```

Comment:
Kan disse "slås sammen" og kalles kanBehandles ? 🤔
... `['GOD_VED', 'BEH_UND_VED', 'BEH_SAK'].includes(oppgavetype)` ... 

Eller blir det litt for upresist/implisitt?

Reply:
Var osikker sjælv! Kan gott prøve å slå sammen til en funksjon :)